### DISCUSSION TITLE
BBTC unusable on SOL?

### DISCUSSION THREAD

**Discussion by maxitimes** [wormhole-foundation/wormhole#344] [Problems / bugs] [answered]
Using Wormhole i transferred some BBTC through wormhole and it appears to have turned it into wBBTC which i don't appear to be able to use anywhere on Solana?  How can i convert this to a suitable BTC on Solana?  If i can't do this, how do i transfer as the drop down in wormhole doesn't have wBBTC as an option to tfr it back? Please help

**Comment by maxitimes** [comment]
I can't see anywhere else to contact wormhole, if i could get some direction it would be appreciated.  

I found the transaction / BTC here
https://solscan.io/token/3UeKTABxz9XexDtyKq646rSQvx8GVpKNwfMoKKfxsTsF#holders

But i have no idea what i can do with it now???

**Comment by leoluk** [comment, marked as answer]
> If i can't do this, how do i transfer as the drop down in wormhole doesn't have wBBTC as an option to tfr it back? Please help

You can contact the authors of the wormholebridge.com UI you're using to request the token to be added to the dropdown: https://github.com/solana-labs/oyster/issues

I don't know if there are any markets for wBBTC - there may be unlisted Serum markets and Saber/Raydium/whatever... pools

**Reply by maxitimes** [reply]
Thank you for the assistance
